test/internal/kubernetes/utils.go

```diff
@@ -38,6 +38,8 @@ const (
 	RetryDelay              = 10 * time.Second
 	DeleteRetryAttempts     = 12
 	DeleteRetryDelay        = 5 * time.Second
+	ShortRetryAttempts      = 8
```

Comment:
can we know why this ShortRetryAttempts is set to 8?

Reply:
ShortRetry* should be used with funtions that fail fast and are short lived. 8 * (250ms + runtime) should come out to a 2.25s - 3s max time.